drivers/tests/UNITTESTS/PwmOut/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set(TEST_NAME pwmout-unittest)
+
+add_executable(${TEST_NAME})
+
+target_compile_definitions(${TEST_NAME} 
+    PRIVATE
+        DEVICE_PWMOUT
+)
+
+target_sources(${TEST_NAME}
+    PRIVATE
+        test_pwmout.cpp
+)
+
+target_link_libraries(${TEST_NAME}
+    PRIVATE
+        mbed-headers
+        mbed-fakes-drivers-pwmout
+        mbed-stubs-hal
+        mbed-stubs-platform      
+        gmock_main
+)
+
+add_test(NAME "${TEST_NAME}" COMMAND ${TEST_NAME})
+
+set_tests_properties(${TEST_NAME} PROPERTIES LABELS "drivers")
```

Comment:
this is new, allows you to run all driver tests with `ctest -L drivers`

if we fix `mbed test` to pass in the `--tests-by-name` param we could use this do run them by name or group like for normal tests

Reply:
This is opened for discussion. If we need `mbed test` to support unittests or we use ctests/cmake directly.

Reply:
I'm in favor of using ctests/cmake directly, they already provide everything needed, no need to put another abstraction layer that will need to be maintained.

Reply:
the `mbed test` is just an side, the label is used by cmake to run tests by group, so you can easily run all drivers, or all LoRa etc. without having to do regex or long lists